arch/lkl/kernel/signal.c

```diff
@@ -50,25 +61,100 @@ void lkl_process_trap(int signr, struct ucontext *uctx)
         /* Handle required signal */
         if(signr == ksig.sig)
         {
+            LKL_TRACE("trap task %p %d\n", current, signr);
             handle_signal(&ksig, uctx);
             break;
         }
     }
 }
 
-void do_signal(struct pt_regs *regs)
+/*
+    Walk to the end of the list and make it point at the new node
+*/
```

Comment:
This comment is for `append_ksignal_node`?

Reply:
Yes, that is what it does.

Reply:
There's some extra space around it then. I assumed it was, but I wasn't completely sure.